<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="download_pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>12</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>다운로드</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URL_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>파일명</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
